### ISSUE TITLE
Delimit synonyms with semicolons, not commas

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/monarch-legacy#1475]
On overview tab of pages like phenotype pages https://monarchinitiative.org/phenotype/HP:0001166, synonyms are delimited with commas, which is odd when there are synonyms that contain commas already. 
<img width="457" alt="screen shot 2017-07-21 at 9 35 57 am" src="https://user-images.githubusercontent.com/3658863/28473074-2768c844-6df8-11e7-9d65-96d3be9946bf.png">

**Comment by pnrobinson** [comment]
I also do not find the ontology hierarchy very enlightening on this page. I wonder if it would not be better to show just the immediate parents and children and to offer a link to a dedicated phenotype browser. We seem to be throwing a kitchen sink at users but not really considering what a typical user would do with the information...

**Comment by jmcmurry** [comment]
Yes, that's part of a whole other conversation. We're hoping to recycle the OLS widget at some point. I thought there was a ticket for that, but I can't find it. Just made one here https://github.com/monarch-initiative/monarch-app/issues/1476
